web-client/src/presenter/actions/JudgeActivityReport/setJudgeActivityReportDataAction.ts

```diff
@@ -4,7 +4,7 @@ export const setJudgeActivityReportDataAction = ({
   props,
   store,
 }: ActionProps) => {
```

Comment:
Not sure if you want to do this or not but if your action requires certain props you can type it! In this case you have casesClosedByJudge, opinions, orders, trialSessions off of props. If you wanted to type what the action looked like it would be:

ActionProps<{casesClosedByJudge: any; opinions: any; orders: any; trialSessions: any;}>

Reply:
Fixed: [20ee9f9](https://github.com/ustaxcourt/ef-cms/pull/3841/commits/20ee9f9f5539be673cc74815f10f001c23c560dd)